server/bin/pbench-server-prep-shim-002.py

```diff
@@ -167,6 +168,7 @@ def process_tb(config, logger, receive_dir, qdir_md5, duplicates, errors):
         # that operation fails it will fail quickly since the file is small.
         try:
             shutil.copy2(tbmd5, dest)
+            selinux.restorecon(dest / tbmd5.name)
```

Comment:
According to my reading, this basically sets default context for the target file. Is it ever possible that the original file in the reception area might have any non-default context? If so, we're throwing it away here.

Reply:
No idea about that. @ndokos

Reply:
If you install this on a VM can we try out a few scenarios to see if this works properly?  Can you devise a test to expose this behavior?